### DISCUSSION TITLE
Züridütsch

### DISCUSSION THREAD

**Discussion by esthicodes** [Estheryu991/SwissGerman_Dictionary#13] [General]
- Abbildig
- Hüt hemmer glernt d'Inverse vonere Matrix zbestimme.

![image](https://user-images.githubusercontent.com/78131082/195310902-ab17e17f-eec0-4ec5-8a76-179c5f668b6a.jpeg)
 

muess alles null ussert diagonale muess 1 sii

![image](https://user-images.githubusercontent.com/78131082/195310541-66feb429-0a31-404b-8cbb-683a2ce97040.jpeg)

https://user-images.githubusercontent.com/78131082/195310956-629ceb4f-8866-4456-ad44-5bed834f989e.MOV

**Comment by esthicodes** [comment]
neubau

**Comment by Lcte21** [comment]
Ich bin reich

**Comment by Lcte21** [comment]
Come back

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/198338763-8a42b016-19d4-4679-b635-e2a05170ba67.jpeg)

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/198339601-2538caa2-73b2-45e8-aeae-aba523bf3106.jpeg)

**Reply by esthicodes** [reply]
![image](https://user-images.githubusercontent.com/78131082/198339888-425003ad-eaa0-4225-87c5-88e4884fd3f7.jpeg)

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/198340058-fda946e6-c0b3-4360-a69f-c8608627e10e.jpeg)

contributor: @LotharCorvega

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/198340262-d80e748f-4e98-4605-ac68-e17602a8f67d.jpeg)
